pcdsdevices/beam_stats.py

```diff
@@ -8,7 +8,6 @@
 from .pv_positioner import PVPositionerDone
 from .signal import AvgSignal
 
-
```

Comment:
Hilariously, after changing here and changing back, all the remains is a flake8 fix.